### PR TITLE
Fix documentation in the default registries.d content.

### PR DESCRIPTION
**Pull request by mtrmac**
… following up on #1750.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@vrothberg PTAL

File: default.yaml [hunk 1]
```diff
@@ -1,26 +1,28 @@
 # This is a default registries.d configuration file.  You may
 # add to this file or create additional files in registries.d/.
 #
-# lookaside: indicates a location that is read and write
-# lookaside-staging: indicates a location that is only for write
+# lookaside: a location simple signing signatures are read from
```
Comment (vrothberg) [inline default.yaml:4]:
```suggestion
# lookaside: a location where simple signing signatures are read from
```
? Same below

Comment (mtrmac) [reply default.yaml:4]: That change was actually wrong, we do write to lookaside: . So I rewrote it completely differently again.

**Review by vrothberg — APPROVED** [review]
LGTM

**Pull request merged**